### PR TITLE
Fix: validate VTT and DR and correctly sort transactions

### LINKED ISSUE #1381: Send tx does not work
when trying to send a value to an address, sometimes does not work even if having enough wits.
It seems like when you try to send a value too high, then it doesn't work, even with a right value.

image

### LINKED ISSUE #1396: Txs not ordered
The txs appear not ordered from the new to the older until a "long" time

image

### LINKED ISSUE #1399: Not enough amount for deploying request
when trying to deploy a dr with not enough amount it should show the error, as when sending a tx. At the moment you can't deploy it but it doesn't show why

### PR DESCRIPTION
**Pull request by gabaldon**
fix #1381 
fix #1399 
fix #1396

### CODE REVIEW

File: src/components/CreateDataRequestForm.vue [hunk 1]
```diff
@@ -78,7 +78,12 @@ export default {
         this.form.witnesses * 2 * this.form.commitAndRevealFee +
         this.form.fee +
         this.form.witnesses * this.form.rewardFee
-      if (totalAmount > this.availableBalance) {
+      const isGreaterThanBalance =
+        parseFloat(
+          standardizeWitUnits(totalAmount, WIT_UNIT.NANO, this.currency),
+        ) > parseFloat(this.availableBalance)
+      console.log(isGreaterThanBalance)
```
Comment (clbartoli) [inline src/components/CreateDataRequestForm.vue:85]:
```suggestion

```

File: src/transactions.js [hunk 2]
```diff
@@ -0,0 +1,1046 @@
+export default [
```
Comment (clbartoli) [inline src/transactions.js:1]: Extra file

File: package.json [hunk 3]
```diff
@@ -106,6 +106,6 @@
   "license": "GPL-3.0",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/witnet/sheikah.git"
+    "url": "git+https://github.com/gabaldon/sheikah.git"
```
Comment (Tommytrg) [inline package.json:109]:
```suggestion
    "url": "git+https://github.com/witnet/sheikah.git"
```

**Review by Tommytrg — APPROVED** [review]

**Review by Tommytrg — APPROVED** [review]

File: src/views/Transactions.vue [hunk 4]
```diff
@@ -40,7 +40,10 @@ export default {
     ...mapState({
       total: state => state.wallet.balance,
       txLabels: state => state.wallet.txLabels,
-      transactions: state => state.wallet.transactions,
+      transactions: state =>
+        state.wallet.transactions.sort(
+          (t1, t2) => new Date(t2.timestamp) - new Date(t1.timestamp),
```
Comment (tmpolaczyk) [inline src/views/Transactions.vue:45]: Timestamps are integers, you can subtract them:

```suggestion
          (t1, t2) => t2.timestamp - t1.timestamp,
```

**Review by clbartoli — APPROVED** [review]

**Pull request merged**